src/Nethermind/Nethermind.Runner.Test/ConfigFilesTests.cs

```diff
@@ -57,6 +66,83 @@ public void Required_config_files_exist(string configWildcard)
             }
         }
 
+        [TestCase(typeof(IInitConfig))]
+        [TestCase(typeof(IBaselineConfig))]
+        [TestCase(typeof(ISyncConfig))]
+        [TestCase(typeof(IAuraConfig))]
+        [TestCase(typeof(IMiningConfig))]
+        [TestCase(typeof(INdmConfig))]
+        [TestCase(typeof(INdmMongoConfig))]
+        [TestCase(typeof(IDbConfig))]
+        [TestCase(typeof(IBloomConfig))]
+        [TestCase(typeof(IEthStatsConfig))]
+        [TestCase(typeof(IGrpcConfig))]
+        [TestCase(typeof(IJsonRpcConfig))]
+        [TestCase(typeof(IKeyStoreConfig))]
+        [TestCase(typeof(IMetricsConfig))]
+        [TestCase(typeof(IStatsConfig))]
+        [TestCase(typeof(IDiscoveryConfig))]
+        [TestCase(typeof(INetworkConfig))]
+        [TestCase(typeof(IKafkaConfig))]
+        [TestCase(typeof(IAnalyticsConfig))]
+        [TestCase(typeof(IHiveConfig))]
+        [TestCase(typeof(ISeqConfig))]
+        [TestCase(typeof(ITxPoolConfig))]
+        [TestCase(typeof(IVaultConfig))]
+        [TestCase(typeof(IWalletConfig))]
+        public void All_default_values_are_correct(Type configType)
+        {
+            ConfigProvider configProvider = new ConfigProvider();
+            PropertyInfo[] properties = configType.GetProperties();
+
+            Type implementationType = configType.Assembly.GetTypes().SingleOrDefault(t => t.IsClass && configType.IsAssignableFrom(t));
+            object instance = Activator.CreateInstance(implementationType);     
+
+
+            foreach(PropertyInfo property in properties)
+            {
+                ConfigItemAttribute attribute = property.GetCustomAttribute<ConfigItemAttribute>();
+                if(attribute == null)
+                {
+                    //there are properties without attribute - we don't pay attention to them 
```

Comment:
is it safe - I know there are some properties that should not have the attribute but what about the ones where we forgot ti add the attribute?